packages/asset-swapper/src/utils/market_operation_utils/constants.ts

```diff
@@ -501,9 +503,11 @@ export const BALANCER_MAX_POOLS_FETCHED = 3;
  */
 // tslint:disable:custom-no-magic-numbers
 export const DEFAULT_GAS_SCHEDULE: Required<FeeSchedule> = {
-    [ERC20BridgeSource.Native]: _fillData => {
-        // const nativeFillData = (_fillData as NativeRfqOrderFillData|NativeLimitOrderFillData)
-        return 100e3;
+    [ERC20BridgeSource.Native]: fillData => {
+        const nativeFillData = fillData as SignedNativeOrder;
+        return nativeFillData && nativeFillData.type === FillQuoteTransformerOrderType.Limit
+            ? PROTOCOL_FEE_MULTIPLIER.plus(100e3).toNumber()
+            : 100e3;
```

Comment:
should be 80 to reflect lower cost of RFQ?

Reply:
synced over slack, should be accurate for RFQ, might leave a note that the limit order cost might need revisiting